added acronym exercise

serialhex opened the pull request:
You never realize how thankful you are for Regex's until you try to do some string parsing without them...

verdammelt commented on exercises/acronym/acronym-test.lisp:
```diff
@@ -0,0 +1,48 @@
+#-xlisp-test (load "acronym")
+
+(defpackage #:acronym-test
+  (:use #:common-lisp #:lisp-unit #:acronym))
```
The build is failing. I think it is because you are specifing that this package uses the `acronym` package. When the tests run the packages defined by the exercises are deleted after the tests run. Some implementations are complaining that `acronym` cannot be deleted because it is in use by `acronym-test`.

verdammelt commented on config.json:
```diff
@@ -3,6 +3,16 @@
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+
```
We should have at least one topic. "Strings" is probably a good one.

serialhex commented:
Added both Strings & Transforming as that's what C# has. Also fixed a merge conflict... I think I borked the history a bit and made it not-so-nice 😿 .

verdammelt commented:
We can do a 'squash and merge' which will remove all history and make a single commit to master.

serialhex commented:
some of my commits were doubled-up, as I saw the potential merge conflict on Github and so locally I rebased, but then my remote was off and then I merged that with resulted in another conflict and blah blah blah... git user fart blah blah...

I reverted and force pushed, so this should be kosher once Travis gives its okay.
